compiler/ast.nim

```diff
@@ -1543,6 +1543,10 @@ proc getStrOrChar*(a: PNode): string =
   case a.kind
   of nkStrLit..nkTripleStrLit: result = a.strVal
   of nkCharLit..nkUInt64Lit: result = $chr(int(a.intVal))
+  of nkBracket:
```

Comment:
Wrong place to fix this, getStrOrChar is supposed to be only used for atomic AST nodes.